mne/minimum_norm/time_frequency.py

```diff
@@ -463,10 +460,10 @@ def compute_source_psd(raw, inverse_operator, lambda2=1. / 9., method="dSPM",
         data_fft = fftpack.fft(data)[:, freqs_mask]
         sol = np.dot(K, data_fft)
 
-        if is_free_ori and pick_ori == None:
+        if is_free_ori and pick_ori is None:
             sol = combine_xyz(sol, square=True)
         else:
-            sol = np.abs(sol) ** 2
+            sol = (sol * sol.conj()).real
```

Comment:
I don't think so, `sol` here comes from the dot product of `K` (which I assume is real) with the result from an FFT, which will be complex, so `sol` should be complex